src/input/pointer.js

```diff
@@ -354,6 +354,17 @@
                             }
                         }
                         break;
+
+                    case MOUSE_WHEEL:
+                        // event inside of bounds: trigger the MOUSE_WHEEL callback
+                        if (eventInBounds) {
+                            // trigger the corresponding callback
+                            if (triggerEvent(handlers, e.type, e, e.pointerId)) {
+                                handled = true;
+                                break;
+                            }
+                        }
+                        break;
```

Comment:
Or use `default:` here and delete the `POINTER_DOWN` case. I kind of prefer that, TBH. It may be more future-proof.